pyaedt/generic/spisim.py

```diff
@@ -252,6 +252,8 @@ def compute_erl(
                 fp.write("# {}: {}\n".format(k, k))
                 fp.write("{} = {}\n".format(k, v))
         retries = 3
+        if "PYTEST_CURRENT_TEST" in os.environ:
+            retries = 10
```

Comment:
Why aren't you just increasing `retries` without looking at the environment variables ? It might also be helpful for users no ?

Reply:
@SMoraisAnsys There is a random issue in the unit tests, that we can not reproduce locally. We think it is related to the number of retries. But if for some reason it really fails, we do not want users to wait 10 times until they see the error. Hopefully, this issue only impacts with pytest, the VMs and SPISIM.